src/dlr_relayvm.cc

```diff
@@ -75,6 +75,13 @@ void RelayVMModel::SetupVMModule(const std::vector<DLRModelElem>& model_elems) {
 
   LoadJsonFromString(metadata_data, this->metadata_);
   ValidateDeviceTypeIfExists();
+  // Override allocator.
+  const char* val = std::getenv("DLR_VM_ALLOCATOR");
+  if ((metadata_.count("Model") && metadata_["Model"].count("VMAllocator") &&
+       metadata_["Model"]["VMAllocator"].get<std::string>() == "naive") ||
+      (val != nullptr && std::string(val) == "naive")) {
+    allocator_type_ = tvm::runtime::vm::AllocatorType::kNaive;
```

Comment:
What's the alternative? and what's default?

Reply:
The default is kPooled